docs/pages/technical-explanation.md

```diff
@@ -167,7 +168,7 @@ Taking our route module from before, here are a few small, but useful UX improve
 2. Focus the input when server side form validation fails
 3. Animate in the error messages
 
-```jsx nocopy lines=[4-6,8-12,21,22,28-30]
+```jsx nocopy lines=[4-6,8-12,23,25,30-34]
```

Comment:
Is this intentional? Didn't cross-check the docs site but just double checking since it's not a formatting change.

Reply:
These weren't changed with the removal of `aria-errormessage` from `form`